bundles/org.openhab.binding.coolmasternet/src/main/java/org/openhab/binding/coolmasternet/internal/ControllerHandler.java

```diff
@@ -44,116 +49,164 @@
 /**
  * Bridge to access a CoolMasterNet unit's ASCII protocol via TCP socket.
  *
- * A single CoolMasterNet can be connected to one or more HVAC units, each with a unique UID.
- * These are individual Things inside the bridge.
+ * <p>
+ * A single CoolMasterNet can be connected to one or more HVAC units, each with
+ * a unique UID. Each HVAC is an individual thing inside the bridge.
  *
  * @author Angus Gratton - Initial contribution
  * @author Wouter Born - Fix null pointer exceptions and stop refresh job on update/dispose
  */
 @NonNullByDefault
-public class ControllerHandler extends BaseBridgeHandler {
-    private static final byte LF = "\n".getBytes(StandardCharsets.US_ASCII)[0];
-    private static final byte PROMPT = ">".getBytes(StandardCharsets.US_ASCII)[0];
-    private static final int SINK_TIMEOUT = 1000;
-    private static final int SOCKET_TIMEOUT = 2000;
+public final class ControllerHandler extends BaseBridgeHandler {
+    private static final byte LF = "\n".getBytes(US_ASCII)[0];
```

Comment:
This should be doable now.
```suggestion
    private static final String LF = "\n";
```

Reply:
Done